app/_locales/en/messages.json

```diff
@@ -5259,6 +5267,10 @@
   "save": {
     "message": "Save"
   },
+  "saveOnTradesWithAMetaMaskReferralCode": {
```

Comment:
Minor, would it help to group the related translations together via some standardised naming such as `hyperliquidDialogDescription`, `hyperliquidDialogTitle` etc.

Reply:
This could be helpful. I wasn't sure if the standard for translation naming is to make it reflect the text itself or the context of where the text sits. Do we tend to have a preference?